robotbase/src/main/java/com/team2357/frc2022/subsystems/TurretSubsystem.java

```diff
@@ -124,21 +159,80 @@ public boolean setTurretRotation(double rotation) {
         setClosedLoopEnabled(true);
         m_targetMotorRotations = rotation * m_config.m_turretGearRatio;
         m_pidController.setReference(m_targetMotorRotations, CANSparkMax.ControlType.kSmartMotion);
+        */
         return true;
     }
 
+    public boolean isRotatingToPosition() {
+        return !Double.isNaN(m_targetMotorRotations);
+    }
+
+    public void flip360() {
```

Comment:
Should this be changed to flip to the limits instead of one. Right now this will never do anything because ```setTurretRotation``` will not set a rotation past the limits.